Gems/Multiplayer/Code/Source/AutoGen/AutoComponent_Common.jinja

```diff
@@ -366,6 +412,19 @@ namespace {{ Component.attrib['Namespace'] }}
     {
     }
 {%     if NetworkInputCount > 0 %}
+{% set net_input_parameters_name = [] %}
+{% call (netInput) ParseNetworkInputsExposedToScript(Component) %}
+{%      set net_input_parameters_name = net_input_parameters_name.append(LowerFirst(netInput.attrib['Name'])) %}
+{% endcall %}
+
+    void {{ ControllerName }}::CreateInput([[maybe_unused]] Multiplayer::NetworkInput& input, [[maybe_unused]] float deltaTime)
+    {
+{%     if (GetNetworkInputsExposedToScriptCount(Component) | int) > 0 %}
+        // The following NetworkInputs have been exposed to script: {{ net_input_parameters_name|join(', ') }}.
+        // The base controller will request these inputs from script...
+        {{ ControllerName }}Base::CreateInput(input, deltaTime);
```

Comment:
How about we name the base method something different, like CreateInputFromScript(...). Reason is stuff gets confusing fast in object hierarchies with overloaded methods where different levels in the hierarchy actually have behaviour. Do I call super::Method()? should I call super::Method() before or after or somewhere in-between all my code? etc.. For this reason I always stick to pure virtuals if at all possible, and if it's absolutely necessary to have a method stubbed in a parent class I leave it completely empty so customers are completely clear it's just a stub and doesn't need to be invoked.

Reply:
Great idea! Check out the update.  Now devs wanting to Create/Process by script don't need to ever remember to call a baseclass method from code.  CreateInputFromScript and ProcessInputFromScript is automatically called before the normal CreateInput and ProcessInput.  If the autocomponent is overriden (example, some inputs are handled by script and other inputs by c++) then all devs need to do is remember to create the network inputs not provided by script.  I went with calling Script before C++ because to me C++ should always have the last word.